### PR TITLE
deps: upgrade graphql-language-service-server to ^1.1.1

### PR DESCRIPTION
**Pull request by lostplan**

### CODE REVIEW

**Comment by lostplan** [comment]
Includes three main changes:
- it works without `watchman`
- supports `textDocument/hover`
- supports online schema (endpoint defined in `.graphqlconfig`)

**Comment by stephen** [comment]
this is great, thank you.

File: package.json [hunk 1]
```diff
@@ -16,7 +16,9 @@
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "node ./node_modules/vscode/bin/test"
   },
-  "categories": ["Languages"],
+  "categories": [
+    "Languages"
+  ],
```
[the remark this answers is not in the document]
Comment (lostplan) [reply package.json]: That was changed by `yarn upgrade`. Figured it could stay 🤷‍♂️

**Comment by stephen** [comment]
I wonder if I did something wrong - I see this when uninstalling watchman:
```
Debugger listening on port 6009.
Warning: This is an experimental feature and could change at any time.
To start debugging, open the following URL in Chrome:
    chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=localhost:6009/f7921bb5-0efd-45c3-8434-0519f6216686
Watchman:  Watchman was not found in PATH.  See https://facebook.github.io/watchman/docs/install.html for installation instructions
/Users/stephen/.vscode/extensions/stpn.vscode-graphql-0.1.9/node_modules/graphql-language-service-server/dist/GraphQLCache.js:360
              mtime: Math.trunc(globResult.statCache[filePath].mtime.getTime() / 1000),
                                                                    ^

TypeError: Cannot read property 'getTime' of undefined
    at /Users/stephen/.vscode/extensions/stpn.vscode-graphql-0.1.9/node_modules/graphql-language-service-server/dist/GraphQLCache.js:360:69
    at Array.map (native)
    at Glob.<anonymous> (/Users/stephen/.vscode/extensions/stpn.vscode-graphql-0.1.9/node_modules/graphql-language-service-server/dist/GraphQLCache.js:357:53)
    at emitOne (events.js:101:20)
    at Glob.emit (events.js:191:7)
    at Glob._finish (/Users/stephen/.vscode/extensions/stpn.vscode-graphql-0.1.9/node_modules/glob/glob.js:197:8)
    at done (/Users/stephen/.vscode/extensions/stpn.vscode-graphql-0.1.9/node_modules/glob/glob.js:182:14)
    at Glob._processSimple2 (/Users/stephen/.vscode/extensions/stpn.vscode-graphql-0.1.9/node_modules/glob/glob.js:688:12)
    at /Users/stephen/.vscode/extensions/stpn.vscode-graphql-0.1.9/node_modules/glob/glob.js:676:10
    at Glob._stat2 (/Users/stephen/.vscode/extensions/stpn.vscode-graphql-0.1.9/node_modules/glob/glob.js:772:12)
[Info  - 2:33:26 PM] Connection to server got closed. Server will restart.

```

**Comment by lostplan** [comment]
I've not seen that before. We'll look into it asap

**Comment by mgadda** [comment]
Just speculating here (testing to follow) that `mtime` might be `undefined` when the file at hand has never been modified.

**Comment by mgadda** [comment]
By the way, the message
```
Watchman:  Watchman was not found in PATH.  See https://facebook.github.io/watchman/docs/install.html for installation instructions
```
is expected behavior when watchman is not installed. This is a difficult message to suppress as it turns out.

**Comment by mgadda** [comment]
@stephen i'm having trouble reproducing the issue, can you post your `.graphqlconfig` here?

**Comment by mgadda** [comment]
Okay. I found a case where might happen. If your `.graphqlconfig` looks like this:
```json
{
  "includes": [ "bad/path/to/afile.graphql" ]
}
```

and `bad/path/to/afile.graphql` does not exist (or more specifically, invoking stat on it returns `ENOENT` or `ENOTDIR`, then `globResult.statCache[filePath]` will be `false` instead of a `Stat` object. Hooray for weakly typed languages! 

If, however, your `includes` property value contains globs with wildcards, such as `bad/path/to/**/*.graphql` then npm-glob will return an empty result set (if nothing matches) and no errors occur. I don't yet understand why this happens but my guess is that if npm-glob thinks the thing it's globbing is not actual a glob, it may take a shortcut and stat the pattern directly. 

While I continue to track this down, @stephen it'd be great to know more about what you have in your `includes` and whether that matches up with what is on the filesystem.

**Comment by stephen** [comment]
not a very interesting one:
```json
{
  "schemaPath": "./gqlschema/schema.json"
}
```

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (lostplan) [reply package.json]: Ah whoops! Sorry about that.

**Comment by mgadda** [comment]
All fixed up now. I'll submit a PR to bump the GLS dependency to 1.1.2 momentarily.

**Pull request merged**